docs/reduction-in-locks.md

```diff
@@ -0,0 +1,100 @@
+# Reduced backup lock time
+
+!!! important
+
+    The `--lock-ddl=REDUCED` option is a [tech preview](./glossary.md#tech-preview). Before using this option in production, we recommend that you test restoring production from physical backups in your environment, and also use the alternative backup method for redundancy.
+
+Percona Server for MySQL implemented `LOCK TABLES FOR BACKUP` to block Data Definition Language (DDL) statements on a server. Percona XtraBackup utilizes this lock during the backup process to ensure that any `DDL` event does not corrupt the backup. The lock does not affect the Data Manipulation Language (DML) statements.
+
+## How the backup lock works
+
+With the [`--lock-ddl=ON`](./xtrabackup-option-reference.md/#lock-ddl) option the backup lock is enabled by default from the start of the backup process.
+
+* Set a backup lock on the server to prevent any `DDL` operations to ensure no new tables are created, renamed, or dropped during the backup.
+
+* Copy redo logs from the checkpoint up to the current Log Sequence Number (LSN) and start tracking new entries.
+
+* Identify and copy the `.ibd` files.
+
+* Copy non-InnoDB files.
+
+* Gather a sync point from all engines (InnoDB LSN, binlog, Global Transaction Identifier (GTID), etc.) by querying the log status.
+
+* Stop the redo thread once it copies at least up to sync point.
+
+* Release the backup lock on the server.
+
+If the backup lock is disabled with the [`--lock-ddl=OFF`](./xtrabackup-option-reference.md/#lock-ddl) option, a backup continues while concurrent DDL events are executed. These backups may be invalid and may fail at either the `backup` or the `--prepare` step.
+
+## The `--lock-ddl=REDUCED` option overview
+
+[Percona XtraBackup 8.4.0-2](./release-notes/8.4.0-2.md) adds the [`--lock-ddl=REDUCED`](./xtrabackup-option-reference.md/#lock-ddl) option to reduce the time the server remains locked by `xtrabackup`. Now, you can execute more `DDL` operations concurrently while the backup is in progress. The backup lock is now taken after copying the `.ibd` files and before copying the `non-InnoDB` files.
+
+### Benefits
+
+The `--lock-ddl=REDUCED` option key features are as follows:
+
+* Reduced locking: Rather than holding the backup lock throughout the entire backup process, it is only acquired for a very short period.
+
+* DDL statements: The server remains accessible for `DDL` operations during the backup process.
+
+* Consistency: Consistency is maintained by using the redo log to track file operations. During the backup process, new metadata files are created to ensure consistency. In the `--prepare` phase, these metadata files are processed first to bring the database to a consistent state.
```

Comment:
Still don't like "Consistency: Consistency" - should be reworded to not have repeated words

Reply:
updated